scripts/cnode-helper-scripts/prereqs.sh

```diff
@@ -78,7 +78,7 @@ EOF
   exit 1
 }
 
-while getopts :in:sflcwpt:m:b: opt; do
+while getopts :in:sflcwt:m:b: opt; do
```

Comment:
`[[ -z ${INSTALL_POSTGREST} ]] && INSTALL_POSTGREST='N'`

Should also be removed.